src/corePlugins.js

```diff
@@ -459,6 +459,11 @@ export let variantPlugins = {
   prefersContrastVariants: ({ addVariant }) => {
     addVariant('contrast-more', '@media (prefers-contrast: more)')
     addVariant('contrast-less', '@media (prefers-contrast: less)')
+    addVariant('contrast-custom', '@media (prefers-contrast: custom)')
```

Comment:
It would be useful to also have a variant for `@media (prefers-contrast)` in a boolean context. But I couldn't think of a good name for it? `contrast-specified`?

To quote the [spec](https://drafts.csswg.org/mediaqueries-5/#prefers-contrast):

>it is also common to reduce visual clutter and color complexity in response to both high and low contrast preferences. In that case, it is appropriate to use @media (prefers-contrast) { … } without specifying [more](https://drafts.csswg.org/mediaqueries-5/#valdef-media-prefers-contrast-more) or [less](https://drafts.csswg.org/mediaqueries-5/#valdef-media-prefers-contrast-less), to do things like replacing background images with plain colors, turning off decorative gradients, or replacing border images or box shadows with simple solid borders. As [prefers-contrast: custom](https://drafts.csswg.org/mediaqueries-5/#descdef-media-prefers-contrast)—like prefers-contrast: more or prefers-contrast: less—evaluates to true in a [boolean context](https://drafts.csswg.org/mediaqueries-5/#boolean-context), such simplifications would also benefit users of [forced colors mode](https://drafts.csswg.org/css-color-adjust-1/#forced-colors-mode), even when their colors of choice do not result in a particularly high or low contrast. This is desirable, as the reduced palette enforced by forced colors mode calls for some visual simplification of the page.